packit_service/worker/handlers/distgit.py

```diff
@@ -248,84 +330,29 @@ def run(self) -> TaskResults:
-                    model.set_downstream_pr_url(downstream_pr_url=downstream_pr.url)
-                    self.sync_release_helper.report_status_for_branch(
-                        branch=branch,
-                        description=f"{self.job_name_for_reporting.capitalize()} "
-                        f"finished successfully.",
-                        state=BaseCommitStatus.success,
-                        url=url,
-                    )
-                    model.set_status(status=SyncReleaseTargetStatus.submitted)
-                except AbortSyncRelease:
-                    logger.debug(
-                        f"{self.sync_release_job_type} is being retried because "
-                        "we were not able yet to download the archive. "
-                    )
-                    model.set_status(status=SyncReleaseTargetStatus.retry)
-                    self.sync_release_helper.report_status_for_branch(
-                        branch=branch,
-                        description=f"{self.job_name_for_reporting.capitalize()} is "
-                        f"being retried because "
-                        "we were not able yet to download the archive. ",
-                        state=BaseCommitStatus.pending,
-                        url=url,
-                    )
-                    return TaskResults(
-                        success=True,  # do not create a Sentry issue
-                        details={
-                            "msg": "Not able to download archive. Task will be retried."
-                        },
-                    )
-                except Exception as ex:
-                    logger.debug(f"{self.sync_release_job_type} failed: {ex}")
-                    # eat the exception and continue with the execution
-                    self.sync_release_helper.report_status_for_branch(
-                        branch=branch,
-                        description=f"{self.job_name_for_reporting.capitalize()} failed: {ex}",
-                        state=BaseCommitStatus.failure,
-                        url=url,
-                    )
-                    errors[branch] = str(ex)
-                    model.set_status(status=SyncReleaseTargetStatus.error)
-                    sentry_integration.send_to_sentry(ex)
-                finally:
-                    model.set_finished_time(finished_time=datetime.utcnow())
-                    model.set_logs(collect_packit_logs(buffer=buffer, handler=handler))
+            for model in sync_release_run_model.sync_release_targets:
+                model.set_status(status=SyncReleaseTargetStatus.retry)
+
+            self.sync_release_helper.report_status_to_all(
+                description=f"{self.job_name_for_reporting.capitalize()} is "
+                f"being retried because "
+                "we were not able yet to download the archive. ",
+                state=BaseCommitStatus.pending,
+                url="",
```

Comment:
Not sure if it's ideal, but if there are no archives, the logs don't make much sense either.